src/rules/ResourceQuoteStyle.js

```diff
@@ -136,10 +136,10 @@ class ResourceQuoteStyle extends ResourceRule {
 
         // used in results to show what the expected quote style is
         const targetQuoteStyleExample = (sourceStyle.ascii || sourceStyle.native) ?
-            `${regExps.target.quoteStart}text${regExps.target.quoteEnd}` :
-            `${regExps.target.quoteStartAlt}text${regExps.target.quoteEndAlt}`;
-        
-        // verify that corresponding quote style is present in target 
+            `${regExps.target.quoteStart[0]}text${regExps.target.quoteEnd[0]}` :
+            `${regExps.target.quoteStartAlt[0]}text${regExps.target.quoteEndAlt[0]}`;
```

Comment:
I find the idea of a quote char suddenly becoming a string with multiple allowed quotes confusing. Could we use an array of quote chars instead?

Reply:
This way, you also wouldn't need to suddenly stuff context-less regexp escape chars in the quotes string